### PR TITLE
Add service objectives page

### PR DESCRIPTION
**Pull request by choldgraf**
This adds a rough service level objectives page to our documentation. The goal is to lay down some basic guidelines for what we strive for in the hub service. It also makes explicit our commitment to protecting the team's time over weekends and holiday periods (and thus is related to https://github.com/2i2c-org/meta/issues/257)

It is not meant to be comprehensive, and is intentionally not extremely detailed. It makes explicit that this is not a legal promise, but our team's *objectives*.

### CODE REVIEW

File: about/service-objectives.md
Comment (damianavila) [inline about/service-objectives.md:23]: This is a really good point I was going to raise, nice to see it already written.

File: about/service-objectives.md
Comment (damianavila) [inline about/service-objectives.md:39]: Great note, again!

**Review by damianavila — APPROVED** [review]
I really like the content on this PR!

**Review by colliand — APPROVED** [review]

File: about/service-objectives.md
Comment (sgibson91) [inline about/service-objectives.md:3]:
```suggestion
This page describes the **Service Level Objectives** (SLOs) of 2i2c's infrastructure services[^slos].
```

**Review by sgibson91 — APPROVED** [review]

**Comment by choldgraf** [comment]
I'd love to get @yuvipanda's 👍 on this one before we merge, since he's got the most experience with SLO-style workflows. Anything major we should change here Yuvi?

**Comment by choldgraf** [comment]
I'll plan to merge this in the next day unless somebody objects, thanks all for the feedback.

**Pull request merged**
